subsys/bluetooth/controller/Kconfig

```diff
@@ -344,6 +344,13 @@ config BT_CTLR_SDC_PERIODIC_ADV_RSP_RX_BUFFER_COUNT
 	  until the host is able to pull the periodic advertising response reports.
 
 	  A value of 0 will disable reception of subevent responses.
+
+config BT_CTLR_SDC_PERIODIC_ADV_RSP_FAILURE_REPORTING_ENABLE
+	int "Failure reporting for the PAwR advertiser"
+	range 0 1
+	default 0
```

Comment:
```suggestion
	bool "Failure reporting for the PAwR advertiser"
```

Reply:
If it's a bool, it's either set or not set. In that case I would have to add some ifdefs setting some auxiliary defines which would then be used to set the config value, which adds some more code.

Also we're considering adding more values as the feature defines another type of failure status which hasn't been implemented [yet]

Reply:
Would it not be better to then add separate bool-kconfigs for those? Instead of having to set the kconfig to some bitfield value.
```kconfig
config A
    bool "a"

config B
    bool "b"
```

```c
uint8_t bitfield = (IS_ENABLED(CONFIG_A) << 0) | (IS_ENABLED(CONFIG_B) << 1)
```